lib/app/routes/exercises.rb

```diff
@@ -1,3 +1,4 @@
+require 'sourceclassifier'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.